### PR TITLE
Add tags page and tags linking.

### PR DESCRIPTION
**Pull request by Hiw**

### CODE REVIEW

**Comment by Hiw** [comment]
[Issue 6: Images by Tag](https://trello.com/c/uakm0GDp/367-issue-6-images-by-tag)

File: app/views/images/tags.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1><%= @tag %></h1>
+<% if @all_images.count > 0 then %>
+<% @all_images.each do |image| %>
+  <img src='<%= image.url %>'/>
+  <br /> <br />
+<% end %>
```
Comment (swarajpatel) [inline app/views/images/tags.html.erb:6]: minor, need indention

Comment (Hiw) [reply app/views/images/tags.html.erb]: added!

**Review by swarajpatel — APPROVED** [review]
👍

**Pull request merged**